src/internal/connector/sharepoint/data_collections.go

```diff
@@ -71,17 +80,53 @@ func DataCollections(
 				if err != nil {
 					return nil, support.WrapAndAppend(site, err, errs)
 				}
-
-				collections = append(collections, spcs...)
 			}
 
+			collections = append(collections, spcs...)
+
 			foldersComplete <- struct{}{}
 		}
 	}
 
 	return collections, errs
 }
 
+func collectLists(
+	ctx context.Context,
+	serv graph.Servicer,
+	tenantID, siteID string,
+	scope selectors.SharePointScope,
+	updater statusUpdater,
+	ctrlOpts control.Options,
+) ([]data.Collection, error) {
+	logger.Ctx(ctx).With("site", siteID).Debug("Creating SharePoint List Collections")
+
+	if scope.Matches(selectors.SharePointSite, siteID) {
+		spcs := make([]data.Collection, 0)
+
+		dir, err := path.Builder{}.Append(siteID).
+			ToDataLayerSharePointPath(
+				tenantID,
+				siteID,
+				path.ListsCategory,
+				false)
```

Comment:
as long as we are consistent with how we name things going forward this is fine

Reply:
actually, should we be using the list name in this path? Otherwise all the items for all lists will land in the same directory and we also won't know what name each list had originally

Reply:
Each list would end up in the same directory with this method. Each list would be an item in that directory.  Is the desire to have a collection per list?
For example
```go
dir, err := path.Builder{}.Append(listID).ToDataLayerSharePointPath(tenantID, siteID, path.ListCategory, true)
```
Note: this would be true as this would be the item and not the directory. @ashmrtn

Reply:
Resolved [9b3af77](https://github.com/alcionai/corso/pull/1788/commits/9b3af77c07de84819f531f1425078cd5b5a89e3b)